src/main/scala/uk/gov/nationalarchives/tdr/api/service/ValidateFileMetadataService.scala

```diff
@@ -86,7 +87,6 @@ class ValidateFileMetadataService(
         _ <- fileStatusRepository.deleteFileStatus(fileIds, Set(ClosureMetadata, DescriptiveMetadata))
         _ <- fileStatusRepository.addFileStatuses(additionalMetadataStatuses)
```

Comment:
So the return type of the method remains the same, return the result of the `fileStatusRepository.addFileStatuses` method call:
```
...
      rows <- fileStatusRepository.addFileStatuses(additionalMetadataStatuses)
   } yield rows
...
```